core/src/com/unciv/logic/map/tile/TileImprovementFunctions.kt

```diff
@@ -314,24 +314,30 @@ class TileImprovementFunctions(val tile: Tile) {
         var stats = Stats()
         for (unique in tile.getTerrainMatchingUniques(UniqueType.ProductionBonusWhenRemoved)) {
             stats.add(unique.stats)
+            if (stats.isEmpty()) return
         }
         val ruleset = civ.gameInfo.ruleset
-        val choppingYieldsIncreaseWithGameProgress = ruleset.modOptions.constants.choppingYieldsIncreaseWithGameProgress
+        val choppingYieldsIncreaseWithGameProgress =
+            ruleset.modOptions.constants.choppingYieldsIncreaseWithGameProgress
         // Civ6 yields increase with game progression: https://www.reddit.com/r/civ/comments/gvx44v/comment/fsrifc2/
         if (choppingYieldsIncreaseWithGameProgress) {
-            val gameProgress = max(civ.tech.researchedTechnologies.size.toFloat() / ruleset.technologies.size, civ.policies.adoptedPolicies.size.toFloat() / ruleset.policies.size)
+            val gameProgress = max(
+                civ.tech.researchedTechnologies.size.toFloat() / ruleset.technologies.size,
+                civ.policies.adoptedPolicies.size.toFloat() / ruleset.policies.size
+            )
             stats *= (1 + 9 * gameProgress)
         }
-        if (distance != 1) stats *= (6-distance)/4
-        if (tile.owningCity == null || tile.owningCity!!.civ != civ) stats *= 2/3
+        if (distance > 5) stats *= 0
```

Comment:
Do you mean it provides no stats? Then you should return here to avoid empty notifications

Reply:
It should? I remember seeing notifications for 0 production from chopping a faraway forest.

Reply:
That's a silly notification, let's avoid those